Articles/Blog/2023-03-08-productivity-practices-and-tools.md

```diff
@@ -0,0 +1,155 @@
+
+#### Tools to consider
+
+If your work life is many short tasks that come and go quickly, consider a white board.
+In addition, consider [Apple Reminders](https://support.apple.com/en-us/HT205890), [Google Keep](https://keep.google.com/), [Trello](https://trello.com/), [Jira](https://www.atlassian.com/software/jira), [TaskWarrior](https://taskwarrior.org/), [Omnifocus](https://www.omnigroup.com/omnifocus), [Things](https://culturedcode.com/things/), [Due](https://www.dueapp.com/), [GitHub Projects](https://github.blog/2022-07-21-tips-tricks-for-using-github-projects-for-personal-productivity/).
+
+### Time Tracking
+
+The last key part of this trio of practices is time tracking.
+That is, keeping track of what you _actually_ spend your time doing. 
+The more frequently you switch tasks and/or are subjected to interruptions, the more challenging time tracking can be.
+A simple practice uses tally marks for each quarter hour (15 minutes) next to task names/ids.
+
+For example, suppose you plan to spend 4 hours on task A in the morning and 4 hours on task B in the afternoon.
+Midway through your morning, a collaborator asks you to quickly review an abstract and bio before it is submitted.
+After spending 15 minutes on this new task C, you take a bathroom break.
+On your way back, a user pulls you into their office asking for help and you spend 25 minutes with them (new task D).
+Your four hour morning on task A has now become 3 hours and 20 minutes.
+
+<table>
+<tr><th>Planned work</th><th>Interruptions</th></tr>
+<tr><td>
+
+Task | Time Tallies | Total
+:--- | :---: | ---:
+Task A | \|\|\|\| &nbsp; \|\|\|\| &nbsp; \|\|\|\| &nbsp; \| | 3.25
+Task B | \|\|\|\| &nbsp; \|\|\|\| &nbsp; \|\|\|\| &nbsp; \|\|\|\| | 4
+**Total**||**7.25**
+
+</td><td>
+
+Task | Time Tallies | Total
+:--- | :---: | ---:
+Task C | \| | 0.25
+Task D | \|\| | 0.5
+**Total**||**0.75**
+
+</td></tr> </table>
+
+With experience, interruption volume can even be predicted and planned accordingly.
+Alternatively, interruptions can sometimes be managed by simply being wholly unavailable (e.g. in your office not answering calls or email, etc.) except at prescribed times.
+
+When reviewing time logs, ask yourself if time could or should have been spent more productivly.
+Why did that happened?
+What could be automated/delegated?
+Don't be surprised if you wind up being surprised what you actually spend your time on.
+
+#### Tools to consider
+
+There are automatic and manual time tracking tools.
+Manual time tracking can be straightforward as the example above shows.
+Automatic tools track application usage and, if configured properly, an account for the same application being used for different purposes.
+Some time tracking tools provide informative ways to visualize how time was spent as a summary.
+
+Consider [ActivityWatch](https://activitywatch.net/), [TimeWarrior](https://timewarrior.net/).
```

Comment:
Activity watch is automatic, time warrior is a manual time tracking tool

Reply:
fixed in 0b3d8418f7b16f4e59df755364ba2746fbc94d5f